### PR TITLE
PUBD-142 Add direct manage URL and modal for the manage button

### PR DESCRIPTION
**Pull request by hardyoyo**
Adds a new CMS field for configuring a direct manage URL for journals. This URL is used in creating the link in the "manage submissions" modal, if it is present.

### CODE REVIEW

File: app/server.rb [hunk 1]
```diff
@@ -720,7 +720,7 @@ def flattenDepts(ids, a=[])
   else
     unit = $unitsHash[ids]
     unitAttrs = JSON.parse(unit.attrs)
-    a << {"id" => unit.id, "name" => unit.name, "directSubmit" => unitAttrs['directSubmit']} unless unit.type != 'oru'
+    a << {"id" => unit.id, "name" => unit.name, "directSubmit" => unitAttrs['directSubmit'], "directManage" => unitAttrs['directManage'],} unless unit.type != 'oru'
```
Comment (myucekul) [inline app/server.rb:723]: Minor - one extra comma - line 723

File: app/unitPages.rb [hunk 2]
```diff
@@ -632,6 +634,7 @@ def getUnitProfile(unit, attrs)
   
   attrs['directSubmit'] and profile[:directSubmit] = attrs['directSubmit']
   attrs['directSubmitURL'] and profile[:directSubmitURL] = attrs['directSubmitURL']
+  attrs['directManageURL'] and profile[:directManageURL] = attrs['directManageURL']
```
Comment (myucekul) [inline app/unitPages.rb:637]: Do we need directManage attrs in addition to directManageURL here?

File: dev/puma.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 port ENV['PUMA_PORT']
 workers ENV['PUMA_WORKERS']
 threads 0, ENV['PUMA_THREADS']
-worker_boot_timeout 120
-worker_timeout 120 # the default of 60 is usually hit on first startup, since this is a dev instance, we can wait
+worker_boot_timeout 220
```
Comment (myucekul) [inline dev/puma.rb:4]: This change is not related to directManage. Right?

File: app/jsx/components/SubheaderComp.jsx [hunk 4]
```diff
@@ -98,11 +99,12 @@ class SubheaderComp extends React.Component {
                 />)
     }
 
+    let manageWizard = null
     let manageButton = <button id="wizardlyManage" className="o-button__3" onClick={(event)=>{
                                this.setState({manageModalOpen:true})
                                event.preventDefault()} } >
                                 Manage<span className="c-subheader__button-fragment">Submissions</span></button>
-    let manageWizard = (<WizardInertComp showModal={this.state.manageModalOpen}
+    manageWizard = (<WizardInertComp showModal={this.state.manageModalOpen}
```
[the remark this answers is not in the document]
Comment (hardyoyo) [reply app/jsx/components/SubheaderComp.jsx:107]: see line 102, the manageWizard is created and set to null with a `let` up there

File: dev/puma.rb (hunk 3 above)
Comment (hardyoyo) [reply dev/puma.rb:4]: you're correct, I'll take this out.

File: app/unitPages.rb (hunk 2 above)
Comment (hardyoyo) [reply app/unitPages.rb:637]: not sure, I was copying the directSubmitURL code
